packages/salesforcedx-vscode-core/src/constants.ts

```diff
@@ -6,8 +6,8 @@
  */
 
 export const SFDX_PROJECT_FILE = 'sfdx-project.json';
-export const SFDX_CONFIG_FILE = 'sfdx-config.json';
-export const SFDX_FOLDER = '.sfdx';
+export const SFDX_CONFIG_FILE = 'config.json';
+export const SFDX_FOLDER = '.sf';
```

Comment:
These are used by WorkspaceContextUtil to listen for config changes.  Ideally, the same functionality would be implemented without watching these files directly since there is no guarantee that they will remain in the same place or continue to have the same data structure.  

I think we should probably keep them the same for now ('sfdx-config.json' and '.sfdx') and create a follow up work item to remove the direct reference to these files.